easybuild/easyconfigs/g/GROMACS/GROMACS-2016.3-foss-2017a.eb

```diff
@@ -28,6 +27,11 @@ toolchainopts = {'openmp': True, 'usempi': True}
 
 source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
 sources = [SOURCELOWER_TAR_GZ]
+checksums = ['7bf00e74a9d38b7cef9356141d20e4ba9387289cbbfd4d11be479ef932d77d27']
+
+dependencies = [
+    ('hwloc', '1.11.8'),
```

Comment:
this should be `1.11.6`, since there's no `hwloc` easyconfig available for `1.11.8` that's compatible with `foss/2017a` but there is for `11.1.6`, i.e. `hwloc-1.11.6-GCC-6.3.0-2.28.eb`

Reply:
My bad, I looked it up in the wrong toolchain. Fixing it.